Sources/RoutingKit/TrieRouter.swift

```diff
@@ -15,11 +28,29 @@ public final class TrieRouter<Output>: Router, CustomStringConvertible {
         case caseInsensitive
     }
 
-    /// Configured options such as case-sensitivity.
-    public var options: Set<ConfigurationOption>
+    private var state: State
 
     /// The root node.
-    private let root: Node
+    private var root: Node {
+        self.state.root
+    }
```

Comment:
Any reason not to just elide this accessor entirely? Especially since you don't even use it anyway (such as in `description`).